examples/shallow_water_Bickley_jet.jl

```diff
@@ -103,7 +103,7 @@ v = vh / h
 compute!(ω)
 
 ## Copy mean vorticity to a new field
-ωⁱ = Field{Face, Face, Nothing}(model.grid)
+ωⁱ = Field((Face, Face, Nothing), model.grid)
```

Comment:
Why?

Reply:
I thought there was a problem with the constructor but in the end, there wasn't. Why do we have this construction anyways?